src/XMakeBuildEngine/Resources/Constants.cs

```diff
@@ -210,7 +210,7 @@ internal static long AssemblyTimestamp
     internal static class AvailableStaticMethods
     {
         /// <summary>
-        /// Static methods that are allowed in constants. Key = Type or Type::Method, Value = AssemblyQualifiedTypeName (where null = mscorlib)
+        /// Static methods that are allowed in constants. Key = Type or Type::Method, Value = Tuple of AssemblyQualifiedTypeName (where null = mscorlib) or the actual type object
```

Comment:
Ah, I see. If we were using code contracts I'd whine and want that as its own class with postconditions but oh well!